src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -1,9 +1,11 @@
+
 package core.mate.academy.model;
 
 /**
  * Add some custom fields that could be only in Truck
  * Do not remove no-field constructor
  */
+
 public class Truck extends Machine {
```

Comment:
Don't forget about access modifiers for your custom fields in `Truck` class. You should define fields with private access modifiers and provide public getters and setters for accessing them.